.github/ISSUE_TEMPLATE/workshop.md

```diff
@@ -15,36 +15,32 @@ assignees: ''
 
 # Tasks
 
-More detailed instructions for specific steps here: <https://osf.io/tzmhp/wiki/Workshops/>
+More detailed instructions for specific steps here: <https://cct-datascience.github.io/group-procedures/workshops.html>
 
 ## Organizing the event
 
--   [ ] Identify presenter, title, description
--   [ ] Draft PR with notes or links to materials in [CALS-workshops repo](https://github.com/cct-datascience/CALS-workshops)
--   [ ] Create a Zoom meeting with registration ([instructions & settings](https://osf.io/tzmhp/wiki/Workshops/))
--   [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
--   [ ] Advertise
-    -   [ ] Edit upcoming workshops section of [group website workshops page](https://datascience.cals.arizona.edu/workshops)
-    -   [ ] Add to Data Science Institute calendar <https://datascience.arizona.edu/calendar>
-    -   [ ] Send out announcement to mailing list
-    -   [ ] Twitter
+- [ ] Identify presenter, title, description
+- [ ] Draft PR with notes or links to materials in [workshop-wednesdays repo](https://github.com/cct-datascience/workshop-wednesdays)
+- [ ] Create a Zoom meeting with registration ([instructions & settings](https://cct-datascience.github.io/group-procedures/workshops.html#creating-a-template-for-workshop-zoom-meetings))
+- [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
+- [ ] Advertise
+  - [ ] Add an event to the [group website workshops page](https://datascience.cals.arizona.edu/workshops)
+  - [ ] Add to Data Science Institute calendar <https://datascience.arizona.edu/calendar>
+  - [ ] Include announcement in the next mailing list email
+  - [ ] resBazAZ and UA Data Science Slack
 
 ## Before workshop
 
--   [ ] Provide homework / background reading as appropriate (installation, references, etc.)
--   [ ] Send out reminders
--   [ ] Create pre- and post-workshop surveys
--   [ ] Send out pre-workshop survey [WIP]
-
-## During Event
-
--   [ ] Provide time for post-workshop survey
+- [ ] Provide homework / background reading as appropriate (installation, references, etc.)
+- [ ] Send out reminders
 
 ## Followup
```

Comment:
```suggestion
## Follow Up
```

Reply:
Actually "Follow-up" is correct if it's a noun 😜